test/specs/openapi/test_examples.py

```diff
@@ -84,7 +152,7 @@ def test_get_request_body_examples(endpoint):
 
 
 def test_get_static_parameters_from_examples(endpoint):
-    static_parameters_list = examples.get_static_parameters_from_examples(endpoint, "examples")
+    static_parameters_list = examples.get_static_params_from_examples(endpoint, "examples")
```

Comment:
Could you, please, rename this function back to `get_static_parameters_from_examples`? I'd prefer slightly more verbose naming